src/pages/K8s.vue

```diff
@@ -75,6 +91,64 @@ export default {
     cannotReset() {
       return (this.state !== K8s.State.STARTED && this.state !== K8s.State.READY);
     },
+
+    invalidMemoryValueReason() {
```

Comment:
Shouldn't this live in `<SystemPreferences>` instead? It's already handling the min/avail stuff, right?

Reply:
This is why I want to combine branches 60-... and 89-... into one branch -- then those validations live only in SystemPreferences, where they act as a guard on updating prefs.